crates/jstz_proto/src/operation.rs

```diff
@@ -140,6 +140,45 @@ pub mod external {
         pub amount: Amount,
         pub reciever: Address,
     }
+
+    #[derive(Debug, Serialize, Deserialize, PartialEq, Eq)]
+    pub struct FaDeposit {
+        // Inbox message id is unique to each message and
+        // suitable as a nonce
+        pub inbox_id: u32,
+        // Amount to deposit
+        pub amount: Amount,
+        // Final deposit receiver address
+        pub receiver: Address,
+        // Optional proxy contract
+        pub proxy_smart_function: Option<Address>,
+        // Ticket hash
+        // Note: Can't use TicketHash (which is a typesafe wrapper around
+        // Blake2b) because it's not Serializable. Type needs to derive Serializable
+        // because it will be a child of ExternalOperation which derives Serializable.
+        // TODO: Revisit if Operation/ExternalOperation needs to be Serialiazable.
```

Comment:
Presumable we will always need to be `Operation`s and `Receipt`s to be serialisable?

Reply:
Why do `Operation`s need to be serialisable?

Reply:
Edit: If I remove De/Serialize derives from `ExternalOperation`, build passes so it can be safely removed. Wdyt?

Reply:
Yeah, I think removing the `De`/`Serialize` derives from `Message` and `ExternalOperation` should be safe

Reply:
Done